salt/_states/metalk8s_kubernetes.py

```diff
@@ -1887,3 +1887,33 @@ def apiservice_absent(name, **kwargs):
     }
 
     return ret
+
+def storageclass_present(
+    name, provisioner, reclaim_policy, volume_binding_mode, mount_options,
+    parameters, **kwargs
+):
+    ret = {'name': name,
+           'changes': {},
+           'result': False,
+           'comment': ''}
+
+    storageclass= __salt__['metalk8s_kubernetes.show_storageclass'](
+        name, **kwargs
+    )
+    if storageclass is None:
+        if __opts__['test']:
+            ret['result'] = None
+            ret['comment'] = 'The StorageClass is going to be created'
+            return ret
+
+        res = __salt__['metalk8s_kubernetes.create_storageclass'](
+            name=name, mount_options=mount_options, parameters=parameters,
+            provisioner=provisioner, reclaim_policy=reclaim_policy,
+            volume_binding_mode=volume_binding_mode, **kwargs
+        ),
+        ret['result'] = True
+        ret['changes'][name] = {'old': {}, 'new': res}
+    else:
+        ret['result'] = True if not __opts__['test'] else None
```

Comment:
Minor: not agree with this one. 
Global salt guideline
Test mode enabled:
- Failure -> result = False
- Nothing to do -> result = True
- Something will be changed -> result = None

So here you should always return True as all good

Reply:
Ok, will do. Thanks for the explanation.

(We probably want to fix `namespace_present` and `serviceaccount_present` but that's probably target an older branch so won't be done here).